supervisord.conf

```diff
@@ -1,29 +1,29 @@
 [program:goofys]
-command = /home/ubuntu/goofys-latest -o allow_other -o nonempty -f %(ENV_AWS_NOTEBOOK_BUCKET)s /home/ubuntu/data/%(ENV_AWS_NOTEBOOK_BUCKET)s
+command = goofys -o allow_other -o nonempty -f jupyterhub-fourfront-notebooks /home/ubuntu/data/jupyterhub-fourfront-notebooks
```

Comment:
Won't this be something we'll have to change back if we make an orchestrated version?